conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, CMake, tools
+
+
+class GoogleBenchmarkConan(ConanFile):
+    name = "benchmark"
+    version = "1.4.1"
+    description = "A microbenchmark support library."
+    url = "https://github.com/google/benchmark"
+    homepage = "https://github.com/google/benchmark"
+    license = "Apache-2.0"
+    settings = "arch", "build_type", "compiler", "os"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_exceptions": [True, False],
+        "enable_lto": [True, False],
+        "enable_testing": [True, False],
+        "enable_gtest_tests": [True, False]
+    }
+    default_options = "shared=False", "fPIC=True", "enable_exceptions=True", "enable_lto=False", "enable_testing=False", "enable_gtest_tests=False"
+    exports_sources = ["*"]
+    generators = "cmake"
+
+    build_subfolder = "."
```

Comment:
Does `"."` mean that it will be built in the source tree?
It should be built in some new subdirectory, e.g. `build`.

Reply:
Conan is creating working and package directories in it's own Conan directory. The original source origin will never be modified.

Reply:
![benchmark](https://user-images.githubusercontent.com/1593194/43282482-8d4f4b78-9116-11e8-85b7-df7cfe45e0f1.PNG)